path_topic1: 'trajectory'
path_topic2: 'path'
cmd_topic: 'cmd_vel'
global_frame: 'local_map_lidar'
base_frame: 'base_link'
odometry_topic: 'lol_odom'
status_topic: 'control_status'
max_v: 0.55
min_v: 0
acc_v: 0.08
max_w: 0.3
acc_w: 0.2
correct_orient_at_last_point: false



